File: 09501.cpp
#include <iostream>
using namespace std;

int main() {
	int T; cin >> T;

	int N; double D;
	for (int i = 0; i < T; i++) {
		cin >> N >> D;

		double v, f, c;
		int cnt = 0;
		for (int j = 0; j < N; j++) {
			cin >> v >> f >> c;

			double time = D / v;
			double available_time = f / c;
			if (available_time >= time) cnt++;
		}

		cout << cnt << endl;
	}
}

File: 09501.py
for i in range(int(input())):
    N, D = map(int, input().split())

    cnt = 0
    for j in range(N):
        v, f, c = map(int, input().split())

        time = D / v
        available_time = f / c
        if available_time >= time: cnt += 1

    print(cnt)
